examples/grpc/client.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const config = require('./setup');
+
+/**
+ * The trace instance needs to be initialized first, if you want to enable
+ * automatic tracing for built-in plugins (HTTP in this case).
+ */
+config.setupTracerAndExporters('grpc-client-service');
+
+const grpc = require('grpc');
+
+const messages = require('./helloworld_pb');
+const services = require('./helloworld_grpc_pb');
+const DEFAULT_PORT = 50051;
+const tracer = opentelemetry.getTracer();
+
+/** A function which makes requests and handles response. */
+function main(port) {
```

Comment:
Do we need to pass port here?

Reply:
No - moved to just use local var instead